warehouse/core/src/main/java/datawave/mr/bulk/BulkIteratorEnvironment.java

```diff
@@ -69,4 +71,14 @@ public SortedKeyValueIterator<Key,Value> reserveMapFileReader(String mapFileName
         return null;
     }
 
+    @Override
+    public ServiceEnvironment getServiceEnv() {
+        return null;
+    }
+
+    @Override
+    public PluginEnvironment getPluginEnv() {
+        return null;
```

Comment:
In Accumulo 2, we're supposed to be fetching PluginEnv instead of caching Configuration objects (which we have done a few times in the past). I suspect having this return null will become a problem when iterators are migrated to caching the PluginEnv (like we did for ConfigurableAgeOffFilter). 

I think we might be able to construct a PluginEnvironment using the conf. 

If this works, we can also start here but I do think we might hit issues eventually.

Reply:
I think that work will definitely need to get done at some point, preferably sooner rather than later. This merge request is just to get bypass accumulo working again.